apps/src/code-studio/signup.js

```diff
@@ -57,8 +59,11 @@ window.SignupManager = function (options) {
     $("#schoolname-block").hide();
 
     // Show correct terms below form.
-    $("#teacher-terms").hide();
     $("#student-terms").fadeIn();
+    $("#teacher-terms").hide();
+
+    // Implicitly accept terms of service for students
+    $("#user_terms_of_service_version").prop('checked', true);
```

Comment:
In the bottom we say that by creating an account as a student you are accepting the TOS and PP, so I think they should have the latest TOS version.